.ui/pgCart.pgx

```diff
@@ -47,49 +50,46 @@
 				"backgroundColor": "rgba( 255, 255, 255, 1 )",
```

Comment:
we shouldn't be using user props inside page

Reply:
done.